src/meta-data/airqo_api.py

```diff
@@ -0,0 +1,75 @@
+import json
+
+import requests
+
+from config import Config
+
+
+class AirQoApi:
+    @staticmethod
+    def remove_suffix(string: str, suffix):
+        if string.endswith(suffix):
+            return string[: -len(suffix)]
+        else:
+            return string[:]
+
+    def __init__(self):
+        self.AIRQO_BASE_URL = AirQoApi.remove_suffix(Config.AIRQO_BASE_URL, "/")
+        self.AIRQO_API_KEY = Config.AIRQO_API_KEY
+
+    def update_site_meta_data(self, site_details: dict):
+
+        params = {"tenant": "airqo", "id": site_details.pop("id")}
+
+        for key, value in site_details.copy().items():
+            if value is None:
+                site_details.pop(key)
```

Comment:
@Baalmart the error on staging are because some fields were null. This code snippet removes fields with null values

Reply:
alright